test/edu-benefits/5490/e2e-test.js

```diff
@@ -0,0 +1,162 @@
+const E2eHelpers = require('../../e2e/e2e-helpers');
+const Timeouts = require('../../e2e/timeouts.js');
+const EduHelpers = require('../../e2e/edu-helpers');
+const Edu5490Helpers = require('../../e2e/edu-5490-helpers');
+const testData = require('./schema/maximal-test.json');
+
+
+module.exports = (client, onlyRequiredFields) => {
+  // Cut out early if we're in production.
+  // Remove this once the 5490 is launched.
+  if (global.__BUILDTYPE__ === 'production') {
+    return;
```

Comment:
This doesn't seem to be working.

@raquelromano-gov Is there anything else I need to do for this safeguard to function?